src/tilda-dbus-actions.xml

```diff
@@ -2,5 +2,6 @@
 <node name="/">
     <interface name="com.github.lanoxx.tilda.Actions">
         <method name="Toggle" />
+        <method name="force_toggle" />
```

Comment:
good point, changed

Reply:
I had a chance to review this today and I think the `force` argument is unnecessary. I think I understand why you added it though. When I am trying this (using Gnome Shell), then I observe the following behavior. Please let me know if I am missing anything.

A) The `force` argument is set to `FALSE`:

1. When I manually run the `tilda` command from a shell, then the call will focus the tilda window that receives the call. The reason here is the non-focus pull-up behavior being set to "Focus Terminal" and the tilda window does not have the focus when you are in a different window.
2. When I manually run the `tilda` command from a shell of the tilda window which I want to hide, then its being pulled up. The reason here is that the window currently has the focus when it receives the D-Bus call.
3. When I register a custom shortcut and run the hotkey while the tilda window does not have focus, then it receives the focus.
4. When I register a custom shortcut and run the hotkey while the tilda window has focsu, then it will be pulled up.

B) The `force` argument is set to `TRUE`:
1. In this case the window will always hide, when it receives the D-Bus command, even if it does not currently have the focus. This may be desired, but it bypasses the non-focus pull-up behavior.

I assume you added the force option, because you tested the tilda command from a different window and the target window was not correctly getting pulled up. But maybe I am missing something and you are not running Gnome Shell or you saw a different behavior than I am and the force option is actually needed for you?

Reply:
Ok, never mind. The merge UI was somehow not refreshed and still showed me the old changes. I see you removed it in your latest commit.